firmware/steering/tests/features/checking_faults.feature

```diff
@@ -9,10 +9,10 @@ Feature: Checking for faults
   Scenario: A sensor becomes permanently disconnected
     Given steering control is enabled
 
-    When a sensor becomes permanently disconnected
+    When a sensor is grounded for 200 ms
 
     Then control should be disabled
-    And a fault report should be published
+    And a fault report should be published with origin ID 1
```

Comment:
Not sure if we can but maybe use the enum here so that the test output is more meaningful in that it shows what the origin ID in context to message type rather than a seemingly arbitrary number?

Reply:
Would you prefer `And a fault report should be published originating from the throttle`?

Reply:
I've removed this abstraction leakage from the tests, see the latest commit on this PR.

Reply:
I like the idea of making sure the origin ID is correct just was wondering if it could be clearer of what it is rather than a variable. Just test readability.